--- dup_time.py.orig	2015-10-26 16:27:59.153178600 +0200
+++ dup_time.py	2015-10-26 16:25:45.625178600 +0200
@@ -28,6 +28,7 @@
 import re
 import calendar
 from duplicity import globals
+from datetime import datetime
 
 
 class TimeException(Exception):
@@ -155,7 +156,10 @@
 
 def timetopretty(timeinseconds):
     """Return pretty version of time"""
-    return time.asctime(time.localtime(timeinseconds))
+
+    isotime = datetime.isoformat(datetime.fromtimestamp(timeinseconds), '.')
+    return isotime
+    #return time.asctime(time.localtime(timeinseconds))
 
 
 def stringtopretty(timestring):
