--- duplicity.orig	2015-10-26 16:28:48.053178600 +0200
+++ duplicity	2015-10-26 16:25:34.705178600 +0200
@@ -693,8 +693,9 @@
     path_iter = diffdir.get_combined_path_iter(sig_chain.get_fileobjs(time))
     for path in path_iter:
         if path.difftype != "deleted":
-            user_info = u"%s %s" % (dup_time.timetopretty(path.getmtime()),
-                                    util.ufn(path.get_relative_path()))
+            user_info = u"\0%s %s %s\0" % (path.type,
+                                           dup_time.timetopretty(path.getmtime()),
+                                           util.ufn(path.get_relative_path()))
             log_info = "%s %s %s" % (dup_time.timetostring(path.getmtime()),
                                      util.escape(path.get_relative_path()),
                                      path.type)
