=== modified file 'bin/duplicity'
--- bin/duplicity	2015-10-31 14:52:22 +0000
+++ bin/duplicity	2015-11-05 15:13:21 +0000
@@ -696,8 +696,9 @@
     path_iter = diffdir.get_combined_path_iter(sig_chain.get_fileobjs(time))
     for path in path_iter:
         if path.difftype != "deleted":
-            user_info = u"%s %s" % (dup_time.timetopretty(path.getmtime()),
-                                    util.ufn(path.get_relative_path()))
+            user_info = u"\0%s %s %s\0" % (path.type,
+                                           dup_time.timetopretty(path.getmtime()),
+                                           util.ufn(path.get_relative_path()))
             log_info = "%s %s %s" % (dup_time.timetostring(path.getmtime()),
                                      util.escape(path.get_relative_path()),
                                      path.type)

=== modified file 'duplicity/dup_time.py'
--- duplicity/dup_time.py	2015-02-01 17:37:37 +0000
+++ duplicity/dup_time.py	2015-11-05 15:14:06 +0000
@@ -28,6 +28,7 @@
 import re
 import calendar
 from duplicity import globals
+from datetime import datetime
 
 
 class TimeException(Exception):
@@ -155,7 +156,10 @@
 
 def timetopretty(timeinseconds):
     """Return pretty version of time"""
-    return time.asctime(time.localtime(timeinseconds))
+
+    isotime = datetime.isoformat(datetime.fromtimestamp(timeinseconds), '.')
+    return isotime
+    #return time.asctime(time.localtime(timeinseconds))
 
 
 def stringtopretty(timestring):

